packages/wdio-local-runner/src/transformStream.js

```diff
@@ -1,5 +1,11 @@
 import { Transform } from 'stream'
 
+const debuggerMessages = [
+    'Debugger listening on',
+    'Debugger attached',
+    'Waiting for the debugger'
+]
```

Comment:
Also this should move into https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-local-runner/src/constants.js